vcd/resource_vcd_org_user.go

```diff
@@ -319,9 +320,10 @@ func resourceVcdOrgUserImport(d *schema.ResourceData, meta interface{}) ([]*sche
 
 	user, err := adminOrg.GetUserByName(userName, false)
 	if err != nil {
-		return nil, govcd.ErrorEntityNotFound
+		return nil, fmt.Errorf("[user import] error retrieving user %s: %s", userName, err)
 	}
 
+	_ = d.Set("org", orgName)
```

Comment:
the org name is mandatory for the import command. It can't be implicit.

Reply:
Yes for import you need to write down orgname. What I am saying that after import, state file will have org name and user configuring imported resource according diff, has to write down org, he don't choice to inherit.

Reply:
This is a problem for most of our resources. If you have inherited the top entity, after import you will need to adjust the script, because you can't import without providing the full chain of names in the hierarchy.
We could change this behavior by modifying all import functions and supply the optional hierarchy names from the provider.

Reply:
Let's leave it for next release.